realm-jni/src/io_realm_internal_SharedGroup.cpp

```diff
@@ -83,15 +83,32 @@ JNIEXPORT jlong JNICALL Java_io_realm_internal_SharedGroup_nativeCreate(
 }
 
 JNIEXPORT jlong JNICALL Java_io_realm_internal_SharedGroup_createNativeWithImplicitTransactions
-  (JNIEnv* env, jobject, jlong native_replication_ptr, jbyteArray keyArray)
+  (JNIEnv* env, jobject, jlong native_replication_ptr, jint durability, jbyteArray keyArray)
```

Comment:
To avoid the long list of params, the both functions are kept for now. Add one more static function to share common code.